erts/configure

```diff
@@ -7330,7 +7330,7 @@ fi
 done
 
 if test -z "$EX_DOC"; then
-  EX_DOC="$ERL_TOP/make/ex_doc_wrapper"
+  EX_DOC="$ERL_TOP/make/ex_doc_wrapper.erl"
```

Comment:
TBH I am unsure what the convention is for writing escript files. When I open and `erl` file inside this source directory I still get all my LSP niceties so I just used `.erl`